xarray/core/coordinates.py

```diff
@@ -259,19 +291,15 @@ def _update_coords(self, coords):
     def variables(self):
         return Frozen(self._data._coords)
 
-    def _to_dataset(self, shallow_copy=True):
```

Comment:
I think removing this should be fine...

Reply:
I searched and there's nothing invoking it across the whole codebase

Reply:
Nice